app/forms/notification_preferences_form.rb

```diff
@@ -4,6 +4,7 @@ class NotificationPreferencesForm < Reform::Form
   attr_reader :current_user, :user_ability
 
   delegate :update_attributes,    to: :model
+  delegate :update,               to: :model
```

Comment:
Probably line 6 (`delegate :update_attributes`) can be deleted now?

Reply:
I think so. Done.